CMakeLists.txt

```diff
@@ -94,6 +94,8 @@ cmake_dependent_option(IREE_ENABLE_COMPILATION_BENCHMARKS "Builds IREE benchmark
 
 option(IREE_BUILD_MICROBENCHMARKS "Builds IREE microbenchmark suites." OFF)
 
+option(IREE_BUILD_E2E_TEST_ARTIFACTS "Builds IREE E2E test artifacts suite generated by the e2e test framework." OFF)
```

Comment:
> You could leave the old option for a bit and use it to set the new one. That way people will be able to easily reuse build directories. I think this would work:

It works. Thanks for the detailed commit suggestions